internal/controller/kustomization_wait_test.go

```diff
@@ -297,6 +299,96 @@ parameters:
 			return apierrors.IsNotFound(err)
 		}, timeout, time.Second).Should(BeTrue())
 	})
+
+	t.Run("processes new revision after failed reconciliation", func(t *testing.T) {
```

Comment:
What's the point of this test? the normal wait behaviour is covered above.

Reply:
I was just scared to break something cause I never contributed before. I'll delete it.